tests/lib/retry.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+snap_install() {
+    local PARAMS=$@
```

Comment:
It would be clearer IMO if we use the values directly in the code instead of declaring those variables.